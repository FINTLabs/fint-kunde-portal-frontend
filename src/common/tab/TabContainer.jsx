import React from "react";
import PropTypes from "prop-types";
import Typography from "@mui/material/Typography";

class TabContainer extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <Typography
        component="div"
        style={{ padding: 8 * 3 }}
      >
        {this.props.children}
      </Typography>
    );
  }
}

TabContainer.propTypes = {
  children: PropTypes.node.isRequired
};

export default TabContainer;
