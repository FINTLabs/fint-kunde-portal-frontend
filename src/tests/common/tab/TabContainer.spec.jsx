import React from "react";
import ReactDOM from "react-dom";
import { create } from "react-test-renderer";
import TabContainer from "../../../common/tab/TabContainer";

describe("TabContainer", () => {
  const compoent = create(
    <TabContainer dir="left">
      <div />
    </TabContainer>
  );
  it("should match snapshot", () => {
    expect(compoent.toJSON).toMatchSnapshot();
  });
});
