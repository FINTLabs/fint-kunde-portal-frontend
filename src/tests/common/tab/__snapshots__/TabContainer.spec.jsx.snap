// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContainer should match snapshot 1`] = `[Function]`;
